<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestsequenceSettingsWidget</class>
 <widget class="QWidget" name="TestsequenceSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Текстовые поля</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Навигация</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="1">
           <widget class="QLineEdit" name="scriptNavigationEdit">
            <property name="text">
             <string>Скрипты</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="testsequenceNavigationEdit">
            <property name="text">
             <string>Тестовые последовательности</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Тестовые последовательности</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Скрипты</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="scriptEditorPathLabel">
         <property name="text">
          <string>Путь до редактора скриптов:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="pathEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="pathBrowserButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Исполнитель теста</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="executorWindowNameLabel">
            <property name="text">
             <string>Название окна исполнителя т.п.</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="executorWindowNameEdit">
            <property name="text">
             <string>Исполнение тестовой последовательности</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="reportHtmlTabNameLabel">
            <property name="text">
             <string>Название вкладки отчета html</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="reportTabNameLabel">
            <property name="text">
             <string>Название вкладки отчета</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="chooseReportTabNameLabel">
            <property name="text">
             <string>Название вкладки выбора отчета</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="reportHtmlTabNameEdit">
            <property name="text">
             <string>Отчет HTML</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="reportTabNameEdit">
            <property name="text">
             <string>Отчет</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="chooseReportTabNameEdit">
            <property name="text">
             <string>Выбор отчета</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Базовые параметры</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="abortAfterErrorBox">
         <property name="text">
          <string>Прекратить исполнение теста, при наличие ошибки</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
